.github/workflows/update-externals.yml

```diff
@@ -0,0 +1,30 @@
+name: Update external pages
+on:
+  schedule:
+    - cron: '30 1 * * 4'
```

Comment:
Sometimes we do release on Thursday during the day. I think it would be better to run this on Fridays. But does not really matter.

Reply:
This isn't really tied at all to releases. The docs that get pulled from cockpit and bots are taken from  master, not a release. Our release process already updates the website's "guide" (and has done that for pretty much ever).

So "Thursday night" is completely arbitrary. We could also run that more often, even daily.